OpenRA.Mods.Common/Traits/Air/Aircraft.cs

```diff
@@ -172,6 +204,7 @@ public void SetPosition(Actor self, WPos pos)
 			{
 				self.World.ScreenMap.Update(self);
 				self.World.ActorMap.UpdatePosition(self, this);
+				IsAirborne = self.World.Map.DistanceAboveTerrain(CenterPosition).Length != 0;
```

Comment:
`> 0`, just in the off chance that someone makes a subterranean aircraft that doesn't break before this.